src/tests/ftest/io/NvmeIO.yaml

```diff
@@ -0,0 +1,49 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+
+hosts:
+ servers:
+  - boro-A
+  - boro-B
+  - boro-C
+  - boro-D
+ clients:
+  - boro-E
```

Comment:
Just a suggestion: You may add one more client in order to reduce the run time. This seems way too long. It will be better to use extra node rather than keeping all others busy for that long.